docs/coordinates/frames.rst

```diff
@@ -42,11 +42,17 @@ is first created::
     >>> FK5()  # uses default equinox
     <FK5 Frame (equinox=J2000.000)>
 
-The specific names of attributes available for a particular frame (and
-their default values) are available as the class method
+The specific names of attributes available for a particular frame are available as the class method
 ``get_frame_attr_names``::
 
     >>> FK5.get_frame_attr_names()
+    dict_keys(['equinox'])
+
```

Comment:
```suggestion
```

Reply:
Fixed (Note that github didn't pick it up, seems it was confused by the two blank lines going to one but thinking I removed the *wrong* blank line.